public/static/locales/fr/form-builder.json

```diff
@@ -249,13 +249,6 @@
     "forOtherEnquiriesPart2": "pour trouver le ministère ou l'organisme qui peut vous aider.",
     "theGCFormsTeam": "L'équipe de Formulaire GC fournit des services aux personnes qui travaillent au sein du gouvernement. Nous ne pouvons pas donner de conseils généraux au public sur les formulaires que nous hébergeons.",
     "backToForms": "Retourner au Formulaires GC"
-  },
```

Comment:
Not sure why this change is cannot be seen in the pull request `Files changed` section but you have somehow deleted strings that are still being used.

Reply:
See https://github.com/cds-snc/platform-forms-client/pull/1342/commits/a267993f1a53ea331d9e4bbfb2487d80f0d14355#diff-a6e486b021382fa7935492752ec9890787007f55c22fdc5c300ec95e11355a7bL253

Reply:
```
"mainNavAriaLabel": "[FR] Main",

  "defaultElementDescription": {
    "email": "Entrez une adresse email comme name@example.com",
    "phone": "Entrez un numéro de téléphone comme 111-222-3333",
    "date": "Entrez une date comme mm/dd/yyyy",
    "number": "N'entrez que des nombres"
  }
```